src/styles/blocks/header.scss

```diff
@@ -0,0 +1,93 @@
+.header {
```

Comment:
https://prnt.sc/eUWY3pCzwX2c => maybe, it is better to make header like 100vh on the screen